### ISSUE TITLE
Custom Pins In Android not showing up

### ISSUE THREAD

**Issue by albilaga** [TorbenK/TK.CustomMap#152]
I have xamarin forms project and when I try to test it in iOS the CustomPins showing up but not in Android (both in emulator). 

I am add it like this

```
Pins.Add(new TKCustomMapPin
            {
                IsVisible = true,
                ShowCallout = false,
                Subtitle = "ic_point_purple.png",
                Image = ImageSource.FromFile("ic_point_purple.png"),
                Title = "",
                IsDraggable = false,
                Position =
                    new Position(DaruApplication.Current.UserPosition.Latitude,
                        DaruApplication.Current.UserPosition.Longitude)
            });
```

And my Pins already implemented INotifyPropertyChanged. I don't use any custom renderer or any modifications in those two. I am already try using Xamarin.Forms and Xamarin.Forms.Maps. 2.3.2 and 2.3.3. and it still the same

**Comment by JeanCollas** [comment]
Try a clean all :)

**Comment by rachelleahr** [comment]
I'm having the same issue. tried cleaning but it didn't help @JeanCollas

**Comment by JeanCollas** [comment]
@Racheldelman Does the 2.3.3.168 version work?
For me it was working, however the new version (2.3.3.175, that requires changing the "OnElementChanged" signature) did not at first, until I did a clean of all projects.

**Comment by rachelleahr** [comment]
@JeanCollas  i tried downgrading to 2.3.3.168, I still couldn't get it to display

**Comment by JeanCollas** [comment]
@Racheldelman was it working before?

**Comment by rachelleahr** [comment]
@JeanCollas i just started this project. didn't have anything working before

**Comment by JeanCollas** [comment]
What about the sample?

**Comment by JeanCollas** [comment]
Do you use the nugget package or the github sources? What branch?

**Comment by rachelleahr** [comment]
nuget package v1.5

**Comment by JeanCollas** [comment]
Use the github instead, probably much more up to date

**Comment by albilaga** [comment]
How do you add references from github @JeanCollas ? It looks like using .net standard 1.5. I tried to change my PCL class to become .NET standard 1.5 and I can't add any nuget packages (xamarin.forms) If I don't change my PCL class to .NET standard 1.5 I can't add references from TK.CustomMap

**Comment by TorbenK** [comment]
Yes, the new pre-version will target .net standard 1.5. There shouldnt be any problem referencing XF though.

I will release it today on nuget.

**Comment by JeanCollas** [comment]
@albilaga 
If you want to reference from GitHub, what I would recommend is:
1. Fork the project (just click on fork, top right)
2. Download and install Github desktop (official client from github, not too bad even if a bit simple, but easy to use)
3. On your solution add the CustomMap projects (main, android and iOS)
4. Link on your UI project the main TKCM project
5. Link on your iOS project the main and iOS project
6. You guess: Link on your Android project, the main and Android project

Done.
Then you just have to update the sources sometimes

**Comment by andreluiz1987** [comment]
I have same problem project Android, Method OnElementChanged  not called. 
Someone knows how to solve?

**Comment by albilaga** [comment]
Solved in package 1.5.1 pre1
